conf/messages.en

```diff
@@ -40,7 +40,7 @@ nth.18 = nineteenth
 results.firstParagraph.youToldTheCalculator = You told the calculator that you {0} {1}
 results.firstParagraph.children = children
 results.firstParagraph.aChild = one child
-results.firstParagraph.dontHave = don''t have
+results.firstParagraph.dontHave = don’t have
```

Comment:
3 tests in FirstParagraphBuilderSpec are failing due to this change but I am fixing these

Reply:
no they are not due to that change... they were broken before i made a change... i checked

Reply:
Thanks @tyagikumarvineet for flagging this. Tests do break with this change - I just checked the change against master. Tests break because we are using the wrong symbol here, hence the need of writing two, to escape it. Fixing this now.